lib/modules/manager/gradle/artifacts.spec.ts

```diff
@@ -545,4 +548,239 @@ describe('modules/manager/gradle/artifacts', () => {
       },
     ]);
   });
+
+  it('updates verification metadata file', async () => {
```

Comment:
_Nitpick:_ would you please collect the dependency verification related tests in a dedicated component? 

Somewhat like this:
```ts
describe('dependency verification tests', () => {
  it('updates verification metadata file', async () => ...
  it('updates verification metadata and lock file', async () => ...
}
```` 

Since `artifact.ts` now implements a second use-case, it would help distinguishing the tests for verification metadata updates.